### PR TITLE
app stuff

### PR DESCRIPTION
**Pull request by mgray07**
Look at Query and jobs, job_list implementation

### CODE REVIEW

File: flask_backend/app.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 app = Flask(__name__)
 CORS(app)
 
-#app
+#apps
```
Comment (rsmith92) [inline flask_backend/app.py:14]: jobs and job_list LGTM

Comment (rsmith92) [reply flask_backend/app.py:14]: getJobsFromQuery function looks good. May need to do some more testing on request.get once we have the frontend linked.

**Comment by rsmith92** [comment]
The code review process here is a bit messy because we worked on one vsCode share session but wanted to create two different pull requests for the parts we did. In the future we would just work on our own files and make our own pull requests.

File: flask_backend/app.py (hunk 1 above)
Comment (mgray07) [reply flask_backend/app.py:14]: Yes, I was planning on doing that. Thanks for the reminder though

**Pull request merged**